src/gui/properties/speedplotview.cpp

```diff
@@ -82,28 +128,9 @@ void SpeedPlotView::setGraphEnable(GraphID id, bool enable)
 
 void SpeedPlotView::pushPoint(SpeedPlotView::PointData point)
```

Comment:
offtopic: could you change the argement to const reference?
and also to SpeedPlotView::setViewableLastPoints() please.

Reply:
`SpeedPlotView::pushPoint` changed.
Did not understand what did you mean to do with `SpeedPlotView::setViewableLastPoints()` ? This method's argument type is enum, there is no profit in converting it to reference

Reply:
>This method's argument type is enum, there is no profit in converting it to reference

Pardon, misread.